blocks/product-support/product-support.css

```diff
@@ -29,9 +29,10 @@
   height: 100%;
 }
 
-.product-support > div > h2 {
-  text-align: center;
-  margin-bottom: 3rem;
+.product-support > div > h4 > strong {
+  letter-spacing: 4px;
```

Comment:
do we have a var for that?

Reply:
Even better, we already have a class - PTAL again